### ISSUE TITLE
Process navigation - active state should be more visible

### ISSUE THREAD

**Issue by zufrieden** [swiss/styleguide#216]
![screenshot 2014-08-08 16 40 42](https://cloud.githubusercontent.com/assets/451236/3858037/14be3430-1f0a-11e4-8978-1ab15b8b69e1.png)

http://adminch.antistatique.net/navigation_modules_-_content_navigation.html

**Comment by LeBenLeBen** [comment]
This should probably be part of the overall refactoring of this component. See #400
